setup.py

```diff
@@ -15,7 +15,10 @@
 from coalib.misc.BuildManPage import BuildManPage
 
 try:
-    locale.getlocale()
+    lc = locale.getlocale()
+    pf = platform.system()
+    if pf != 'Windows' and lc == (None, None):
```

Comment:
yeah but you have no reason to exclude it. Windows might too produce `(None, None)` and then the error slips through again :+1: